File: csharp/ex041.cs
// Desenvolva um programa que mostre na tela a seguinte contagem:
// 100 95 90 85 80 ... 0 Acabou!


class Ex041
{
  public int counter = 100;

  public void FirstCountdown()
  {
    Console.Write(" "+counter+" ");
    while (counter > 0)
    {
      counter -= 5;
      Console.Write($"{counter} ");
    }
    Console.Write("Acabou!");
  }
}

File: typescript/src/ex041.ts
// Desenvolva um programa que mostre na tela a seguinte contagem:
// 100 95 90 85 80 ... 0 Acabou!

let numbValueFort : number = 105;


while (numbValueFort > 0) {
    numbValueFort -= 5;
    process.stdout.write(numbValueFort + " ");
}
console.log(` Acabou!`);